Style-Guide/Code-Layout-and-Formatting.md

```diff
@@ -127,22 +127,22 @@ More explicit code is more maintainable. While PowerShell allows leaving off the
 
 Usually you will press the `[Tab]` key to indent, but most editors can be configured to insert spaces instead of actual tab characters. For most programming languages and editors (including PowerShell ISE) the default is four spaces, and that's what we recommend. Different teams and projects may have different standards, and when contributing to a project, you should abide by the predominant style, of course.
 
-```powershell
+```PowerShell
 function Test-Code {
-    foreach ($exponent in 1..10) {
-        [Math]::Pow(2, $exponent)
+    foreach ($Exponent in 1..10) {
+        [System.Math]::Pow(2, $Exponent)
```

Comment:
I'll give you this one, since the example below uses the full `[System.Math]` but I'm not a fan of spelling that our for no reason 😕

Reply:
Personally when writing code I would just go with [Math], but the change I made was more concerned with consistency than [Math] vs [System.Math]